docs/guides/tfhers/pregenerated-key.md

```diff
@@ -0,0 +1,16 @@
+# Second Use Case: Pregenerated TFHE-rs keys
+
```

Comment:
This document provides an overview of the second use case for combining TFHE-rs and Concrete, where two parties work with pregenerated TFHE-rs keysets. The focus is on how to perform computations on encrypted TFHE-rs values using Concrete, without the need for a shared secret key.